project/MultiNode.scala

```diff
@@ -100,8 +109,7 @@ object MultiNode extends AutoPlugin {
       (headerCreate in MultiJvm).value
       (compile in MultiJvm).value
     }) ++ headerSettings(MultiJvm) ++ Seq(
-      // only works if I put it here ¯\_(ツ)_/¯
-      compile in MultiJvm := ((compile in MultiJvm).triggeredBy(compile in Test)).value
+      validateCompile := compile.?.all(anyConfigsInThisProject).value
     )
```

Comment:
if we can't make it in `test:compile` we should add `validateCompile` to the new `all*` command aliases

Reply:
:+1: